x-pack/plugins/apm/public/components/app/mobile/transaction_overview/transaction_overview_tabs/stats_list/get_columns.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { RIGHT_ALIGNMENT, EuiText } from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { ValuesType } from 'utility-types';
+import { APIReturnType } from '../../../../../../services/rest/create_call_apm_api';
+import {
+  ChartType,
+  getTimeSeriesColor,
+} from '../../../../../shared/charts/helper/get_timeseries_color';
+import { SparkPlot } from '../../../../../shared/charts/spark_plot';
+import { isTimeComparison } from '../../../../../shared/time_comparison/get_comparison_options';
+import {
+  asMillisecondDuration,
+  asPercent,
+  asTransactionRate,
+} from '../../../../../../../common/utils/formatters';
+import { ITableColumn } from '../../../../../shared/managed_table';
+
+type MobileMainStatisticsByField =
+  APIReturnType<'GET /internal/apm/mobile-services/{serviceName}/main_statistics'>;
+
+type MobileMainStatisticsByFieldItem = ValuesType<
+  MobileMainStatisticsByField['mainStatistics']
+>;
+
+type MobileDetailedStatisticsByField =
+  APIReturnType<'GET /internal/apm/mobile-services/{serviceName}/detailed_statistics'>;
+
+export function getColumns({
+  agentName,
+  detailedStatisticsLoading,
+  detailedStatistics,
+  comparisonEnabled,
+  offset,
+}: {
+  agentName?: string;
+  detailedStatisticsLoading: boolean;
+  detailedStatistics: MobileDetailedStatisticsByField;
+  comparisonEnabled?: boolean;
+  offset?: string;
+}): Array<ITableColumn<MobileMainStatisticsByFieldItem>> {
+  return [
+    // version/device
+    {
+      field: 'name',
+      name: i18n.translate(
+        'xpack.apm.mobile.transactions.overview.table.nameLabel',
```

Comment:
why is this `nameLabel` and the other one are `...ColumnLabel`? Would be good to keep this consistent. Same applies to crash rate.

Reply:
Addressed here https://github.com/elastic/kibana/pull/153044/commits/a79ebaa66ebd32c6ef351b2b7eaddfc3acd60c6c